packages/backend/src/core/DiscoveryWatcher.ts

```diff
@@ -42,6 +42,7 @@ export class DiscoveryWatcher {
 
   start() {
     this.logger.info('Started')
+    this.taskQueue.addToFront(UnixTime.now())
```

Comment:
why this one? we are only running the bot every hour

Reply:
Oupsi dupsi